src/Tizen.NUI/src/internal/Interop/NDalicPINVOKE.cs

```diff
@@ -171,10 +171,7 @@ public static void Set(global::System.Exception e)
                 if (pendingException != null)
                     throw new global::System.ApplicationException("FATAL: An earlier pending exception from unmanaged code was missed and thus not thrown (" + pendingException.ToString() + ")", e);
                 pendingException = e;
-                lock (typeof(NDalicPINVOKE))
-                {
-                    numExceptionsPending++;
-                }
+                numExceptionsPending++;
```

Comment:
I agree to remove `lock(this)` in event pattern. 
but this is not event pattern. please consider to add new dummy object to check the Mutex.

Reply:
I've added a dummy object.